tests/check_sanity.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+bash travis-scripts/run-sanity-tests;
+SANITY=$?
+
+
+if [[ "$SANITY" == "0" ]]; then
+  echo -e "\e[92mPUPPETEER SANITY TESTS OK"
+  exit 0;
+else
+  echo -e "\e[91mPUPPETEER SANITY TESTS FAILED"
+  exit 255;
+fi
```

Comment:
You also can do:
```bash
if [[ $SANITY -ne 0 ]]; then
  echo -e "\e[91mPUPPETEER SANITY TESTS FAILED"
  exit 255;
fi

echo -e "\e[92mPUPPETEER SANITY TESTS OK"
exit 0;
```

Reply:
I know it's a more logical approach, but I always thought it was a little confusing to read, having this last `exit` not conditioned.

Reply:
Commit first suggestions instead if this one
thanks @PierreRambaud